frontend/src/components/eam-configurate-group/components/create-group/create-group.tsx

```diff
@@ -0,0 +1,101 @@
+import { FC } from 'react';
```

Comment:
we should have only one page for it
check the `SC` configurate page

Reply:
but [SC configuration](https://github.com/BinaryStudioAcademy/bsa-winter-2021-2022-bws/blob/development/frontend/src/components/sc-configurate-instance/sc-configurate-instance.tsx) have two page